presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -78,13 +92,39 @@ protected QueryRunner createQueryRunner()
     @Test
     public void testColumnReferencedTwice()
     {
-        ByteBuffer buf = ByteBuffer.allocate(8);
-        buf.putLong(0, 1);
+        ByteBuffer buf = ByteBuffer.allocate(48);
```

Comment:
Use larger values which span all the bytes for needed for given data type, so we are sure we are reading everything.
If you use `1` as test value for 8-byte char we can be reading just a single byte and still not-fail assertion. Sth like `0x1234567890abcdef` will give better coverage.